README.md

```diff
@@ -27,20 +27,29 @@ Dependencies in Debian trixie:
 build-essential meson ninja-build pkg-config qt5-qmake qttools5-dev-tools qtbase5-dev libtermpaint-dev libtuiwidgets-dev libposixsignalmanager-dev
 ```
 
+Installing dependencies in Fedora:
+```
+sudo dnf install gcc gcc-c++ meson ninja-build pkg-config qt5-qtbase-devel qt5-linguist
+```
+
+Fedora do not have packages for [Tui Widgets](https://tuiwidgets.namepad.de/) and [Termpaint](https://termpaint.namepad.de/) so you have to install them from sources.
```

Comment:
I hope this is going to be outdated soon.
Maybe this would be better phrased as a condition ("if your distribution does not have ... in a recent enough version ..." style).